predicators/envs/spot_env.py

```diff
@@ -317,7 +317,8 @@ def _build_observation(self,
 
         # Detect objects.
         object_names_in_view_by_camera = \
-            self._spot_interface.get_objects_in_view_by_camera()
+            self._spot_interface.get_objects_in_view_by_camera\
+                (from_apriltag=CFG.spot_grasp_use_apriltag)
```

Comment:
this doesn't seem quite right because this doesn't have anything to do with grasping, but the setting name has `grasp` in it. and I think this confusion leads to functional issues -- see **